package.yaml

```diff
@@ -1,5 +1,5 @@
 name:                wallet-proxy
-version:             0.16.0
+version:             0.16.1
```

Comment:
Let's splurge on 0.17 since we are expanding the API.

Reply:
Done.